prime-dotnet-webapi/Services/EnrolleeService.cs

```diff
@@ -93,6 +93,39 @@ public async Task<string> GetActiveGpidAsync(Guid userId)
                 .SingleOrDefaultAsync();
         }
 
+        /// <summary>
+        /// Gets the GPID and other details of an Enrollee.
+        /// Returns null if no Enrollee exists with the given UserId or if the Enrollee is in the 'Declined' status
+        /// </summary>
+        /// <param name="userId"></param>
+        public async Task<HpdidLookup> GetActiveGpidDetailAsync(Guid userId)
+        {
+            return await _context.Enrollees
+                .Where(enrollee => enrollee.UserId == userId
+                    && enrollee.CurrentStatus.StatusCode != (int)StatusType.Declined)
+                .Select(e => new HpdidLookup
+                {
+                    Gpid = e.GPID,
+                    Hpdid = e.HPDID,
```

Comment:
Thinking about it more, for this API's client, we should **not** be returning HPDID

Reply:
HPDID has been removed.